src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Controllers/DelegationsController.cs

```diff
@@ -82,6 +85,66 @@ public async Task<ActionResult> Post([FromBody] List<Rule> rules)
             }
         }
 
+        /// <summary>
+        /// Endpoint for retrieving delegated rules between parties
+        /// </summary>
+        /// <response code="400">Bad Request</response>
+        /// <response code="500">Internal Server Error</response>
+        [HttpPost]
+        [Authorize(Policy = AuthzConstants.ALTINNII_AUTHORIZATION)]
+        [Route("authorization/api/v1/[controller]/GetRules")]
+        public async Task<ActionResult<List<Rule>>> GetRules([FromBody] RuleQuery ruleQuery, [FromQuery] bool onlyDirectDelegations = false)
```

Comment:
I would like us to create a wrapper class instead of returning the raw list/array. It is easier to expand a custom class with additional properties than it is to do so with a List<T>(). Imagine for example that we find out that we need paging.

Reply:
Paging will never be relevant for this endpoint - all the rules returned will have to be considered. We've discussed this in the team, and can't see that there ever will be a need to add information to describe the aggregate itself. All the information required to match policymatches in the rulequery is present in each rule. Also, AddRules- and DeletedRules-operations (already merged) already return List<Rules>.